### ISSUE TITLE
pipenv --venv takes 0.5 sec

### ISSUE THREAD

**Issue by jabbalaci** [pypa/pipenv#2483]
`pipenv --venv` is quite slow:

```
$ time pipenv --venv
...
real    0m0.496s
user    0m0.450s
sys     0m0.046s
```

Why is it so slow? Is there a way to speed it up?

**Comment by uranusjr** [comment]
> Why is it so slow?

Because the implementation is not optimal.

> Is there a way to speed it up?

Yes.

> Is there a lot of time available to invest on it?

Not really.

Do lend a hand if you’d like.

**Comment by techalchemy** [comment]
This is fixed in #2485 --

```console
 ~/g/pypa-pipenv       pipenv --version
pipenv, version 2018.7.1
 ~/g/pypa-pipenv       time -p pipenv --venv
/home/hawk/.virtualenvs/pypa-pipenv-PuUpLSqX-3.7
real 0.12
user 0.10
sys 0.02
```

**Comment by jabbalaci** [comment]
Thank you, it's fixed now. Here are my results:

**Before:**

```
$ time pipenv --version
pipenv, version 2018.6.25

real    0m0.486s
user    0m0.456s
sys     0m0.030s
=======================================================================
$ time pipenv --venv
/home/jabba/...

real    0m0.485s
user    0m0.455s
sys     0m0.030s
```

**After:**

```
$ time pipenv --version
pipenv, version 2018.7.1

real    0m0.111s
user    0m0.101s
sys     0m0.010s
=======================================================================
$ time pipenv --venv
/home/jabba/...

real    0m0.108s
user    0m0.102s
sys     0m0.007s
```
